Library/Formula/devd.rb

```diff
@@ -0,0 +1,132 @@
+require "language/go"
+
+class Devd < Formula
+  desc "devd: a local webserver for developers"
```

Comment:
Don't include the formula name at the start of the description. Happens automatically.